docs/tad.md

```diff
@@ -86,40 +86,125 @@ This service also has to contact the recommendation service to retrieve custom j
 
 ### Message Service
 
-- Explain the role of the message services
+The Message Service is responsible for managing messages and message channels within the LinkedOut platform. It handles the creation and retrieval of messages and message channels. By doing so, it allows the user to communicate with employers and directly exchange messages with them.
```

Comment:
Describe a bit more this section: what is a message channel? how to create them?